### PR TITLE
IBAN validtion

### PR DESCRIPTION
**Pull request by minsunny**
https://nanopay.atlassian.net/browse/NP-2857

nanopay : https://github.com/nanoPayinc/NANOPAY/pull/11226

Added IBAN validation from com.google.foam.demos.iban.IBAN so that we can validate IBAN front end

### CODE REVIEW

File: src/foam/nanos/nanos.js [hunk 1]
```diff
@@ -448,5 +448,8 @@ FOAM_FILES([
   { name: "foam/nanos/geocode/GoogleMapsCredentials" },
 
   { name: "com/google/foam/demos/u2/AllViews" },
-  { name: "com/google/foam/demos/u2/MDAllViews" }
+  { name: "com/google/foam/demos/u2/MDAllViews" },
```
Comment (jlhughes) [inline src/foam/nanos/nanos.js:451]: @kgrgreer can we move this foam/u2 ? 
@minsunny what in this are we using?

File: src/foam/nanos/iban/ValidationIBAN.js [hunk 2]
```diff
@@ -0,0 +1,237 @@
+foam.CLASS({
```
Comment (jlhughes) [inline src/foam/nanos/iban/ValidationIBAN.js:7]: header.

File: src/foam/nanos/nanos.js (hunk 1 above)
Comment (minsunny) [reply src/foam/nanos/nanos.js:451]: this was added with this PR https://github.com/foam-framework/foam2/pull/4114

Comment (jlhughes) [reply src/foam/nanos/nanos.js:451]: My point here is that we are dependent on a 'demo' path for production code.  
If there is something in MDAllViews we are using it should move to foam/u2

Comment (kgrgreer) [reply src/foam/nanos/nanos.js:451]: Scott added that, this isn't really part of this PR, she just added a comma. It is just part of the documentation, so I think it's fine.

**Pull request merged**